references/cli/dryrun.go

```diff
@@ -60,11 +61,7 @@ func NewDryRunCommand(c common.Args, ioStreams cmdutil.IOStreams) *cobra.Command
 			return c.SetConfig()
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			velaEnv, err := GetEnv(cmd)
-			if err != nil {
-				return err
-			}
-			buff, err := DryRunApplication(o, c, velaEnv.Namespace)
+			buff, err := DryRunApplication(o, c, namespace)
```

Comment:
Why don't we get the namespace from env like other commands?

Reply:
First we want it to align with kubectl vela plugin. Now I think I misunderstood @wonderflow . Maybe we should add an global option `--namespace`, use NS from env if it's not specified.